graphql_compiler/tests/test_helpers.py

```diff
@@ -267,6 +272,18 @@ def get_schema():
     return schema
 
 
+def generate_schema(graph_client):
+    """Generate schema and type equivalence dict from a pyorient client"""
+    schema_records = graph_client.command(ORIENTDB_SCHEMA_RECORDS_QUERY)
+    schema_data = [x.oRecordData for x in schema_records]
+    type_overrides = {
+        "UniquelyIdentifiable": {"uuid": GraphQLID},
+        "Animal": {"birthday": GraphQLDate},
```

Comment:
@pmantica1, do the `GraphQLDate` and `GraphQLDateTime` fields need type overrides? They should be stored as `Date` and `DateTime` objects in the database, so I thought they'd get auto-detected as such in our schema builder. Is that not the case?